lib/contentdm_translator.rb

```diff
@@ -181,13 +181,18 @@ def self.cdm_record(at_id)
   end
 
   def self.cdm_url_to_iiif(url)
-    matches = url.match(/https?:\/\/(cdm\d+).*collection\/(\w+)\/id\/(\d+)/)
-    if matches
-      server = matches[1]
-      collection = matches[2]
-      record = matches[3]
-      
-      "https://#{server}.contentdm.oclc.org/digital/iiif-info/#{collection}/#{record}/manifest.json"
+    matches = url.match(/https?:\/\/(cdm\d+)(?:.*collection\/(\w+)(?:\/id\/(\d+))?)?/)
+    
+    server = matches[1]
```

Comment:
Will this error if `matches` is false, say if the user puts in a non-URL or bad URL?